apps/condo/bin/sbbol/credentials.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Manages credentials through SBBOL API
+ *
+ * @example Loop through all organizations and refresh tokens:
+ * yarn node apps/condo/bin/sbbol/credentials.js refresh-all-tokens
+ *
+ * @example Change client secret for clientId `1234` that have an old client secret `a1b2c3d4` to new value of `asdf12345`
+ * yarn node apps/condo/bin/sbbol/credentials.js change-client-secret 1234 a1b2c3d4 asdf12345
+ */
+const path = require('path')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+const { values } = require('lodash')
+const { SbbolCredentials } = require('@condo/domains/organization/integrations/sbbol/SbbolCredentials')
+
+
+const COMMAND = {
+    REFRESH_ALL_TOKENS: 'refresh-all-tokens',
+    CHANGE_CLIENT_SECRET: 'change-client-secret',
+}
+
+
+const workerJob = async () => {
+    const [command] = process.argv.slice(2)
+    if (!values(COMMAND).includes(command)) {
+        throw new Error('Wrong command.')
+    }
+
+    const resolved = path.resolve('apps/condo/index.js')
+    const { distDir, keystone, apps } = require(resolved)
+    const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+    // we need only apollo
+    await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+    await keystone.connect()
```

Comment:
you don't need to do it inside the job. It's already done by `tasks.js`

Reply:
Could you please check the `deliveryMessage` example

Reply:
It's a bin for manual calling, not a task.
I've checked `deliveryMessage` and taken an example of getting Keystone from that.

Reply:
The name is confuse me